collections/Hatton/MS_Hatton_donat_1.xml

```diff
@@ -38,42 +37,565 @@
+                           </dimensions>
+                        </layout>
+                     </layoutDesc>
                   </objectDesc>
+                  <handDesc>
+                     <handNote script="cursivaAntiquior">Written in an Anglicana bookhand, seemingly by a single scribe.</handNote>
+                  </handDesc>
                   <decoDesc>
-                     <summary>
-                        <ref target="https://catalog.hathitrust.org/Record/000468709">Pächt and Alexander</ref> iii. 1060:</summary>
-                     <decoNote type="border">Good borders</decoNote>
-                     <decoNote type="decInit">Good initials.</decoNote>
+                     <decoNote type="border">Open with an eight-line initial W on a gold field, bordered with penwork and green acanthus scrollwork (fol. 1r).</decoNote>
+                     <decoNote type="decInit">Textual divisions, either the openings of prologues or tales, are marked by initials of two to four lines with penwork and borders. Versals in blue or gold, with red or blue penwork (<ref target="https://catalog.hathitrust.org/Record/000468709">Pächt and Alexander</ref> iii. 1060).</decoNote>
+                     <decoNote type="rubrication">Rubricated incipits and explicits, running heads, and glosses.</decoNote>
                   </decoDesc>
                   <bindingDesc>
-                     <binding notBefore="1600" notAfter="1700">
-                        <p>17th-cent. brown leather.</p>
+                     <binding notAfter="1700" notBefore="1600">
+                        <p>Fragment from the back of an earlier binding, labelled <q>MSS D Hatton</q>, inside the upper board. The second endleaf (fol. ii) has also been removed from this binding and pasted to a more recent endleaf.</p>
+                     </binding>
+                     <binding notAfter="1800" notBefore="1700">
+                        <p>Eighteenth century, brown suade leather over pasteboard.</p>
                      </binding>
                   </bindingDesc>
                </physDesc>
                <history>
                   <origin>
-                     <origDate calendar="Gregorian" notAfter="1460" notBefore="1450">c. 1450–1460</origDate>
+                     <origDate calendar="Gregorian" notAfter="1460" notBefore="1450">1450s</origDate>
                      <origPlace>
                         <country key="place_7002445">English</country> (dialect East Midland but some Northern forms)</origPlace>
                   </origin>
+                  <provenance>Sixteenth-century inscriptions: <q>William <unclear>Hurst</unclear></q> (fol. 9r), <q>Wyllm <unclear>Shirley</unclear></q> (fol. 124r), <q>Iohn Da</q> (fol. 208r).</provenance>
+                  <provenance><persName key="person_000000" role="fmo">Capt. Charles Hatton</persName>, brother of the <persName key="person_37729288">first Baron Hatton</persName> (inscribed, fol. ii recto); given to the:</provenance>
```

Comment:
I cannot find Charles Hatton in the persons index, and his dates are not immediately obvious, though we have some other correspondence by him and I think he is <https://www.wikidata.org/wiki/Q22998036> (correspondent of Francis Bacon). NB the son of Christopher Hatton is also named Charles Hatton, <https://www.wikidata.org/wiki/Q75762697>.

Reply:
I think (?) it is the person described at pp. 255ff of http://www.northamptonshirerecordsociety.org.uk/eNpp/NppNo18_b.pdf , dates 1635-c.1708

Reply:
Perfect; thank you for finding this!